terraform/root_common.tf

```diff
@@ -156,3 +156,17 @@ module "jenkins_sign_commits_policy" {
   name          = "TDRJenkinsSignCommitsPolicy"
   policy_string = templatefile("./tdr-terraform-modules/iam_policy/templates/jenkins_github_gpg_policy.json.tpl", { account_id = data.aws_caller_identity.current.account_id })
 }
+
+module "notifications_topic" {
+  source      = "./tdr-terraform-modules/sns"
+  common_tags = local.common_tags
+  function    = "notifications"
+  project     = var.project
+  kms_key_arn = module.encryption_key.kms_key_arn
+}
+
+module "jenkins_integration_cloudwatch_agent_policy" {
```

Comment:
Should this have "integration" in the name of the module, as won't this be common for both integration and prod?

Reply:
Good spot. I was originally going to do one for each environment but changed my mind and copied and pasted it in here. I've changed it.